frontend-new/src/App.res

```diff
@@ -26,24 +25,17 @@ let collectBranches = (data: array<GetBenchmarks.t_benchmarks>) => {
   ->Belt.Set.String.toArray
 }
 
-let getTestMetrics = (item: GetBenchmarks.t_benchmarks): array<BenchmarkTest.testMetrics> => {
-  item.json_data
-  ->Belt.Option.getExn
-  ->jsonFieldExn("result", Js.Json.Object)
-  ->Js.Dict.get("results")
-  ->Belt.Option.getExn
-  ->Js.Json.decodeArray
-  ->Belt.Option.getExn
-  ->Belt.Array.map(result => {
-    {
-      BenchmarkTest.name: jsonFieldExn(result, "name", Js.Json.String),
-      metrics: result
-      ->jsonFieldExn("metrics", Js.Json.Object)
-      ->jsDictToMap
-      ->Belt.Map.String.map(v => BenchmarkTest.decodeMetricValue(v)),
-      commit: item.commits,
-    }
-  })
+let getTestMetrics = (item: GetBenchmarks.t_benchmarks): BenchmarkTest.testMetrics => {
+  {
+    BenchmarkTest.name: item.repo_id,
```

Comment:
Just a note: `BenchmarkTest.name` should ideally be mapped to the `benchmarks.test_name` SQL column. I haven't applied the migration that adds this column yet, so let's keep it like this in this PR.